lang/en/quizaccess_seb.php

```diff
@@ -89,3 +89,6 @@
 
 // Capabilities.
 $string['seb:bypassseb'] = 'Bypass the requirement to view quiz in Safe Exam Browser.';
+
+// Quiz access errors.
+$string['invalidkeys'] = "The config key or browser exam keys could not be validated. Please ensure you are using the Safe Exam Browser with correct configuration file";
```

Comment:
We need to check wordings

Reply:
Yeah, especially for the ones user facing. Should I start compiling a list of strings that will need wordings?